src/main/java/org/minions/devfund/noemiguzman/battleship/Ocean.java

```diff
@@ -0,0 +1,187 @@
+package org.minions.devfund.noemiguzman.battleship;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    private static final int SIZE_OCEAN = 20;
+    private static final int NRO_SHIPS = 13;
+    private Ship[][] ships = new Ship[SIZE_OCEAN][SIZE_OCEAN];
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+
+    /**
+     * Creates an empty ocean (fills the ships array with a bunch of EmptySea instances).
+     * Also initializes any game variables, such as how many shots have been fired.
+     */
+    public Ocean() {
+        shotsFired = 0;
+        hitCount = 0;
+        shipsSunk = 0;
+        for (int i = 0; i < SIZE_OCEAN; i++) {
+            for (int j = 0; j < SIZE_OCEAN; j++) {
+                EmptySea emptySea = new EmptySea();
+                emptySea.placeShipAt(i, j, true, this);
+            }
+        }
+    }
+
+    /**
+     * Place all randomly on the (initially empty) ocean.
+     * Place larger ships before smaller ones, or you may end up with no legal place to put a large ship.
+     */
+    public void placeAllShipsRandomly() {
+        Ship[] shipsEmpty =  newShips();
+        for (Ship ship : shipsEmpty) {
+            ship.putRandom(this);
+        }
+    }
+
+    /**
+     * new ships method.
+     * @return ship list
+     */
+    private Ship[] newShips() {
+        Ship[] shipsShips = new Ship[NRO_SHIPS];
+        for (int i = 0; i < NRO_SHIPS; i++) {
+            shipsShips[i] = selectShip(i);
+        }
+        return shipsShips;
+    }
+
+    /**
+     * get a ship.
+     * @param i int
+     * @return Ship object
+     */
+    private Ship selectShip(int i) {
+
+        final int iLC = 4;
+        final int iDES = 7;
+
+        Ship ship1;
+        switch (i) {
+            case 0:  ship1  = new BattleShip();
+                break;
+            case 1:  ship1  = new BattleCruiser();
+                break;
+            case 2: case iLC - 1: ship1 = new Cruiser();
+                break;
+            case iLC: case iLC + 1:  ship1 = new LightCruiser();
+                break;
+            case iDES - 1: case iDES: case iDES + 1: ship1 = new Destroyer();
+                break;
+
+            default: ship1 = new Submarine();
+                break;
+        }
+        return ship1;
```

Comment:
Consider using a factory design patter using a Map with lambdas
to avoid having a switch case.